config/kinto.ini

```diff
@@ -1,35 +1,37 @@
+[server:main]
+use = egg:waitress#main
+host = 0.0.0.0
+port = 8888
+
 [app:main]
 use = egg:kinto
-
-pyramid.debug_notfound = true
-
 kinto.http_scheme = http
-kinto.http_host = localhost:8888
-
-# kinto.project_docs = https://kinto.readthedocs.org/
```

Comment:
Because you do not need to change the doc URL from the configuration. The default values is the right one in the scope of kinto.